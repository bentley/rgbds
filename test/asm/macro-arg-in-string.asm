print: MACRO
	PRINTT "\1"
	PRINTT "\n"
ENDM

	print John "Danger" Smith
	print \\A\nB
	print C\
D
	print E\!F ; illegal character escape


iprint: MACRO
	PRINTT "{\1}"
	PRINTT "\n"
ENDM

s EQUS "hello"
	iprint s
