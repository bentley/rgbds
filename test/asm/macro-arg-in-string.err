ERROR: macro-arg-in-string.asm(10) -> macro-arg-in-string.asm::print(2):
    Illegal character escape '!'
error: Assembly aborted (1 errors)!
