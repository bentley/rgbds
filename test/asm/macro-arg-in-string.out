John "Danger" Smith
\A
B
CD
E\F
hello
